h2/src/test/org/h2/test/db/TestCompatibility.java

```diff
@@ -288,6 +288,14 @@ private void testPostgreSQL() throws SQLException {
                 /* Expected! */
             }
         }
+
+        /* Test that C-style line comments //... are not allowed */
+        try {
+            stat.execute("// Don't do anything, actually");
+            fail("Expected JdbcSQLException, because //... comments are not allowed in PostgreSQL mode");
```

Comment:
H2 test units have `assertThrows()` for such tests.

Reply:
Oh, thank you! Will fix it.